src/freenas/etc/rc.conf.local

```diff
@@ -767,6 +767,8 @@ _gen_conf()
 		echo "mountd_flags=\"${mountdflags}\""
 	done
 
+	echo "inadyn_flags=\"--continue-on-error\""
```

Comment:
Can we put this in /etc/rc.conf, please?

rc.conf.local is mostly for dynamic things, and since that flag is static there is no point. Plus I get chills everytime we touch this file :P